build.gradle

```diff
@@ -109,7 +107,6 @@ dependencies {
     testAnnotationProcessor 'org.projectlombok:lombok:1.18.12'
     testCompileOnly 'org.projectlombok:lombok:1.18.12'
 
-    testImplementation 'org.codehaus.groovy.modules.http-builder:http-builder:0.7.1'
```

Comment:
looks like snyk isn't configured correctly though.  It shouldn't have reported a test implementation vulnerability :confused:

Reply:
afaik that is not something that you are able to configure